tests/tests/test-eth-fee/test-eth-fee-history.ts

```diff
@@ -61,7 +62,7 @@ describeDevMoonbeamAllEthTxTypes("Fee History", (context) => {
 
   it("result length should match spec", async function () {
     this.timeout(100000);
-    let max_fee_per_gas = "0x3B9ACA00";
+    let max_fee_per_gas = "0x"+DEFAULT_TXN_MAX_BASE_FEE.toString(16);
```

Comment:
This test is still failing (specifically, the assert below that expects `result.baseFeePerGas.length` to be `block_count + 1`). I'm not very clear on what the test is doing -- is it obvious to you, @tgmichel?